client.bal

```diff
@@ -115,59 +117,68 @@ public client class Client {
     #                       request as the continuation-token.
     # 
     # + return - If success, returns S3Object[] object, else returns error
-    remote function listObjects(string bucketName, string? delimiter = (), string? encodingType = (), int? maxKeys = (),
-                                string? prefix = (), string? startAfter = (), boolean? fetchOwner = (), 
-                                string? continuationToken = ()) returns @tainted S3Object[]|error {
-        map<string> requestHeaders = {};
-        map<string> queryParamsMap = {};  
-        http:Request request = new;
+    @display {label: "Get objects"}
+    remote function listObjects(@display {label: "Bucket name"} string bucketName,
```

Comment:
If there is a large file in the bucket, how we are going to handle it?

Reply:
We need to handle this in future improvements. Created an issue to track. https://github.com/wso2-enterprise/choreo/issues/3531